src/main.cpp

```diff
@@ -79,7 +79,7 @@ void loop() {
     led_blue.handle();
     led_green.handle();
 
-    // Publish a message every publish interval
+    // Publish a message every publish interva
```

Comment:
Looks like we had a casualty in the latest commit 😛

Reply:
Haha oops!